packages/components/src/divider/stories/index.tsx

```diff
@@ -45,20 +54,23 @@ const VerticalTemplate = ( args ) => {
 	return (
 		<div className={ wrapperClassName }>
```

Comment:
Do you think we should now inline these styles (`style={{ display: 'flex' }}` etc.) so it's clearer in the code snippet (in Docs view) what's going on? I think our initial thinking was that we should demonstrate how to use Emotion in the examples, but on second thought it's probably irrelevant to consumers who don't use Emotion.

<img src="https://user-images.githubusercontent.com/555336/154546191-d076df30-8613-4939-88b5-bb67615aff1b.png" alt="Code snippet in Docs view that is exposing an Emotion classname" width="300">